### PR TITLE
Change uppercase true and false to lowercase in php

### PR DESCRIPTION
**Pull request by mstruebing**
As we changed this convention some time ago we should reflect this in our code :)

### CODE REVIEW

File: Neos.Eel/Classes/AbstractParser.php [hunk 1]
```diff
@@ -159,97 +159,97 @@ function match_StringLiteral ($stack = array()) {
...
 	while(0);
-	if( $_20 === TRUE ) { return $this->finalise($result); }
-	if( $_20 === FALSE) { return FALSE; }
+	if( $_20 === true ) { return $this->finalise($result); }
+	if( $_20 === false) { return false; }
 }
 
 
-/* BooleanLiteral: 'true' | 'TRUE' | 'false' | 'FALSE' */
+/* BooleanLiteral: 'true' | 'true' | 'false' | 'false' */
```
Comment (mstruebing) [inline Neos.Eel/Classes/AbstractParser.php]: Attention here

File: Neos.Flow/Classes/Cli/RequestBuilder.php [hunk 2]
```diff
@@ -274,10 +274,10 @@ protected function getValueOfCurrentCommandLineOption(string $currentArgument, a
                 if ($expectedArgumentType !== 'boolean') {
                     return $possibleValue;
                 }
-                if (array_search($possibleValue, ['on', '1', 'y', 'yes', 'true', 'TRUE']) !== false) {
+                if (array_search($possibleValue, ['on', '1', 'y', 'yes', 'true', 'true']) !== false) {
```
Comment (mstruebing) [inline Neos.Flow/Classes/Cli/RequestBuilder.php]: Attention here

File: Neos.Flow/Classes/Cli/RequestBuilder.php [hunk 3]
```diff
@@ -274,10 +274,10 @@ protected function getValueOfCurrentCommandLineOption(string $currentArgument, a
...
                     return true;
                 }
-                if (array_search($possibleValue, ['off', '0', 'n', 'no', 'false', 'FALSE']) !== false) {
+                if (array_search($possibleValue, ['off', '0', 'n', 'no', 'false', 'false']) !== false) {
```
Comment (mstruebing) [inline Neos.Flow/Classes/Cli/RequestBuilder.php]: Attention here

**Comment by mstruebing** [comment]
I can't reproduce the error locally :/

**Comment by albe** [comment]
Have you tried rebasing and pushing again? I don't see an obvious error that would cause this test failure.

File: Neos.Eel/Classes/AbstractParser.php [hunk 4]
```diff
@@ -189,67 +189,67 @@ function match_BooleanLiteral ($stack = array()) {
...
-				if (( $subres = $this->literal( 'false' ) ) !== FALSE) {
+				if (( $subres = $this->literal( 'false' ) ) !== false) {
 					$result["text"] .= $subres;
-					$_29 = TRUE; break;
+					$_29 = true; break;
 				}
 				$result = $res_26;
 				$this->pos = $pos_26;
-				if (( $subres = $this->literal( 'FALSE' ) ) !== FALSE) {
+				if (( $subres = $this->literal( 'false' ) ) !== false) {
```
Comment (kitsunet) [inline Neos.Eel/Classes/AbstractParser.php]: This change is wrong and might related to the failing tests.

**Comment by mstruebing** [comment]
Reverted the changes on autogenerated parser classes.

**Comment by kitsunet** [comment]
Neos.Eel/Resources/Private/PHP/php-peg/Compiler.php

is also not our code...
It's somewhere from here https://github.com/hafriedlander/php-peg
And we should probably make it a dev dependency or something, but for this change I would also not touch that file.

**Comment by mstruebing** [comment]
Fixed

**Comment by kitsunet** [comment]
But this includes the Abstract and EelParser again?

**Comment by mstruebing** [comment]
A final newline, like it is set in the `.editorconfig`, so I guess it's okay?

**Comment by kitsunet** [comment]
True, I don't care for the newlines, that's fine, but Neos.Eel/Classes/EelParser.php has actual changes.

**Comment by mstruebing** [comment]
Not anymore, sorry, I restored the original file from master, not from 5.0

**Comment by mstruebing** [comment]
ping :)

**Pull request merged**